x/wasm/internal/types/msg_test.go

```diff
@@ -12,22 +10,27 @@ import (
 )
 
 func TestBuilderRegexp(t *testing.T) {
-	cases := []struct {
+	cases := map[string]struct {
 		example string
 		valid   bool
 	}{
-		{"fedora/httpd:version1.0", true},
-		{"cosmwasm-opt:0.6.3", true},
-		{"cosmwasm-opt-:0.6.3", false},
-		{"confio/js-builder-1:test", true},
-		{"confio/.builder-1:manual", false},
+		"normal":                {"fedora/httpd:version1.0", true},
+		"another valid org":     {"confio/js-builder-1:test", true},
+		"no org name":           {"cosmwasm-opt:0.6.3", false},
+		"invalid trailing char": {"cosmwasm-opt-:0.6.3", false},
```

Comment:
This test should have an org. Otherwise it is invalid for two different reasons.

Reply:
Good catch. Fixed test